### PR TITLE
ts: add function on perp position to get notional value

### PR DESCRIPTION
**Pull request by microwavedcola1**

### CODE REVIEW

File: ts/client/src/accounts/mangoAccount.ts [hunk 1]
```diff
@@ -1264,6 +1258,10 @@ export class PerpPosition {
     return this.marketIndex !== PerpPosition.PerpMarketIndexUnset;
   }
 
+  public getNotionalValueUi(perpMarket: PerpMarket): number {
+    return this.getBasePositionUi(perpMarket, true) * perpMarket.uiPrice;
```
[the remark this answers is not in the document]
Comment (ckamm) [reply ts/client/src/accounts/mangoAccount.ts:1262]: 1/ yeah, not taking quote into account is fine. Is there an equivalent function for quote value that can take the event queue into account too?
2/ I think it should be negative if the base position is negative

[the remark this answers is not in the document]
Comment (microwavedcola1) [reply ts/client/src/accounts/mangoAccount.ts:1262]: 1/ and 2/ good, keeping as is then

Sorry could you please elaborate on `is there an equivalent function for quote value that can take the event queue into account too?`

[the remark this answers is not in the document]
Comment (microwavedcola1) [reply ts/client/src/accounts/mangoAccount.ts]: done

File: ts/client/src/accounts/mangoAccount.ts [hunk 2]
```diff
@@ -1283,6 +1277,25 @@ export class PerpPosition {
...
+    }
+
+    const quotePositionUi = toUiDecimalsForQuote(this.quotePositionNative);
+
+    return useEventQueue
+      ? quotePositionUi + perpMarket.quoteLotsToUi(this.takerQuoteLots)
+      : quotePositionUi;
+  }
+
+  public getNotionalValueUi(perpMarket: PerpMarket): number {
```
Comment (ckamm) [inline ts/client/src/accounts/mangoAccount.ts:1295]: could this still pass through `useEventQueue`? Then users can use getquotePosUi/getNotationalValueUi either with or without event queue.

**Review by ckamm — APPROVED** [review]

**Pull request merged**